test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -822,6 +822,27 @@ Describe "Invoke-WebRequest tests" -Tags "Feature" {
         $response.Content.Method | Should -Be $redirectedMethod
     }
 
+    It "Validates Invoke-WebRequest handles responses without Location header correctly for requests with Authorization header and redirect: <redirectType>" -TestCases $redirectTests {
```

Comment:
It is good for comment but very long for header. Please make shorter header and add the comment.
Below too.

Reply:
@TravisEz13 I get conflicting feedback on that. For now, I think it's best if this follows the code around it. When I refactor the tests, I will address this globally.

Reply:
I added the whitespace and shortened the test description by one word. It's now shorter than the surrounding tests. I agree that `Validates` is redundant, but as all the other tests include it, I include it for consistency. 

I don't agree that a test header should be short. It really must describe the test in a good way and removing anything (apart from `Validates`) would change the meaning of the test.

@markekraus already stated that he wants to cleanup the tests and he can consistently remove `Validates` for all tests.

Reply:
I'm ok with delaying the changes to a later refactoring.